site/en/docs/privacy-sandbox/fledge/index.md

```diff
@@ -234,15 +234,15 @@ relevant to the interest group, and a URL for JavaScript used in bidding. Intere
 {% Aside %}
 
 Interest groups are stored in the browser and can make bids in FLEDGE
-in-browser auctions, but aren't programmatically 'readable'. In other words
-there's no `navigator.getAdInterestGroups()`, as this would expose cross-site
+in-browser auctions, but aren't programmatically 'readable.' In other words,
+there's nothing like navigator.getAdInterestGroups(), as this would expose cross-site
```

Comment:
I don't think navigator.getAdInterestGroups() should be in code font since it doesn't actually exist